packages/preferences/src/browser/views/preference-editor-widget.ts

```diff
@@ -230,7 +230,7 @@ export class PreferencesEditorWidget extends BaseWidget implements StatefulWidge
         }
     };
 
-    onScroll = debounce(this.doOnScroll.bind(this), 50, { leading: false, trailing: true });
+    onScroll = throttle(this.doOnScroll.bind(this), 50);
```

Comment:
Runs the scroll logic more often to skip fewer nodes.

Reply:
Nice, feels much better